op.c

```diff
@@ -8926,6 +8957,8 @@ Perl_utilize(pTHX_ int aver, I32 floor, OP *version, OP *idop, OP *arg)
         use_version = sv_2mortal(new_version(use_version));
         S_enable_feature_bundle(aTHX_ use_version);
 
+        U16 shortver = S_extract_shortver(aTHX_ use_version);
+
         /* If a version >= 5.11.0 is requested, strictures are on by default! */
         if (vcmp(use_version,
                  sv_2mortal(upg_version(newSVnv(5.011000), FALSE))) >= 0) {
```

Comment:
Seeing as we now have the `shortver` here, we could avoid constructing a version object just for the comparison.